README.md

```diff
@@ -97,9 +97,15 @@ Complete all of these tasks on your local home machine.
 * Streisand requires a BSD, Linux, or macOS system. As of now, Windows is not supported. All of the following commands should be run inside a Terminal session.
 * Python 2.7 is required. This comes standard on macOS, and is the default on almost all Linux and BSD distributions as well. If your distribution packages Python 3 instead, you will need to install version 2.7 in order for Streisand to work properly.
 * Make sure an SSH key is present in ~/.ssh/id\_rsa.pub.
+  * SSH keys use public-key cryptography to prove your identity to a server or service.
```

Comment:
I'm not sure that someone unfamiliar with SSH keys will find much clarity in a reference to public-key cryptography. In counter-balance, we don't have a ton of space in a README to really teach a whole course on anything.

How about:
"SSH keys are a more secure alternative to passwords that allow you to prove your identity to a server or service built on public key cryptography."?

I think that maintains the intent of your edit but also makes it clear that conceptually a new user can think of it as a more secure password without really losing much.